src/edu/washington/escience/myria/operator/agg/AggregatorFactory.java

```diff
@@ -30,4 +31,11 @@
    */
   @Nonnull
   Aggregator get(Schema inputSchema) throws DbException;
+  /**
+   * @param inputSchema
+   * @param pyFuncReg
+   * @return
+   * @throws DbException
+   */
+  Aggregator get(Schema inputSchema, PythonFunctionRegistrar pyFuncReg) throws DbException;
```

Comment:
aggregator is not an operator, which is why i need to pass it along ( for UserDefinedAggregator ) otherwise we could get from the underlying operator. and since I changed the interface -- i made it uniform everywhere.